tests/test_checks.py

```diff
@@ -100,6 +104,9 @@ def test_month(self, tasmin_series):
         miss = checks.missing_any(ts, freq='YS', month=8)
         np.testing.assert_equal(miss, [True])
 
+        with pytest.raises(ValueError):
```

Comment:
if you also want to validate the error message, `pytest.raises(ValueError, match=r"No data for selected period."):`.  Useful if you raise the same Exception but for various different reasons.

Reply:
fixed